pkg/registry/core/serviceaccount/storage/storage_test.go

```diff
@@ -38,7 +38,7 @@ func newStorage(t *testing.T) (*REST, *etcdtesting.EtcdTestServer) {
 		DeleteCollectionWorkers: 1,
 		ResourcePrefix:          "serviceaccounts",
 	}
-	return NewREST(restOptions), server
+	return NewREST(restOptions, nil, nil, nil), server
```

Comment:
Why not introduce a new method like `NewRESTWithToken()` and keep the old function as-is?

Reply:
We shouldn't be creating this REST without a token subresource.

Reply:
agree on keeping a single signature, though in cases where there was no signing key, or the feature gate was disabled, I'd expect the subresource to be absent (not listed in discovery)